ArduCopter/system.cpp

```diff
@@ -412,11 +412,7 @@ void Copter::update_auto_armed()
         if(flightmode->has_manual_throttle() && ap.throttle_zero && !failsafe.radio) {
             set_auto_armed(false);
         }
-        // if helicopters are on the ground, and the motor is switched off, auto-armed should be false
-        // so that rotor runup is checked again before attempting to take-off
-        if(ap.land_complete && motors->get_spool_state() != AP_Motors::SpoolState::THROTTLE_UNLIMITED && ap.using_interlock) {
-            set_auto_armed(false);
-        }
+
```

Comment:
I'm a little surprised removing this makes any difference...

Reply:
@rmackay9 From what I remember, this was the primary cause for heli's not being allowed to spool up in guided or auto modes.  this would always cause auto_armed to be false disabling spool up from ground idle for heli's

Reply:
I agree this line should be removed.  It causes problems in other ways as well, in particular a copter with interlock setup on a switch cannot run a mission which involves landing and taking off again because auto-armed goes false so it gets stuck.